src/index.coffee

```diff
@@ -9,7 +9,7 @@ module.exports = (env, callback) ->
 
     getPages: (contents) ->
       env.helpers.contents.list(contents).filter (content) ->
-        content instanceof env.plugins.MarkdownPage
+        content instanceof env.plugins.MarkdownPage unless content.metadata?.noindex
```

Comment:
If all objects returned by `getPages` will have a `metadata` property, why does the view need to check for `if page.metadata?`